### PR TITLE
Preferred language

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by gberaudo** [comment]
See #116.

File: c2corg_api/models/user.py [hunk 1]
```diff
@@ -66,6 +67,8 @@ class User(Base):
     validation_nonce_expire = Column(DateTime, nullable=True, unique=False)
     _password = Column('password', String(255), nullable=False)
 
+    lang = Column(String(2), nullable=False, default='fr')
```
Comment (tsauerwein) [inline c2corg_api/models/user.py]: This should be a foreign-key:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/document.py#L223

**Comment by tsauerwein** [comment]
Looking good! Only one small comment.

File: c2corg_api/models/user.py (hunk 1 above)
Comment (gberaudo) [inline c2corg_api/models/user.py]: Thanks, done.

**Comment by gberaudo** [comment]
Thanks for the review.

**Pull request merged**